redirect/admin.py

```diff
@@ -58,9 +58,24 @@ class RedirectRuleAdmin(admin.ModelAdmin):
     )
 
 
+class DomainNameInline(admin.TabularInline):
+    model = DomainName
+    extra = 0
+    min_num = 2
+
+
+@admin.display(description="Domain names")
+def domain_names(self):
+    return ", ".join(self.names.values_list("name", flat=True))
```

Comment:
Suggesting prefetching the names in DomainAdmin

```python
class DomainAdmin(admin.ModelAdmin):
    def get_queryset(self, request):
        return super().get_queryset(request).prefetch_related("names")
```

Might be prefetch doesn't work for values_list ( :shrug: ) or it requires some extra black magic. So may have to change this to use `.all()`:

```suggestion
    return ", ".join(domain_name.name for domain_name in self.names.all())
```

Reply:
Seems to work with `values_list`.